.rubocop.yml

```diff
@@ -45,6 +45,11 @@ Style/AlignParameters:
 Style/ClosingParenthesisIndentation:
   Enabled: false
 
+Style/DoubleNegation:
```

Comment:
It doesn't make sense for me...

If you are breaking this rule, in special, in some place, I don't see any reason to enable it. So, you could only disable it for all project.

Reply:
It is enabled by default. Project wise, so the way to add exception is like so.
Because this rule is to prevent double negative to be used in checking for falsy values and objects give that would be redundant given that could just use `.nil?`.
Since the case where the double negative is being used doesn't fit the original description of the rule, just ignored on that case in specific while maintaining the rule for the rest of the system.